.github/workflows/delegator_generic_slsa3.yml

```diff
@@ -244,3 +244,15 @@ jobs:
         with:
           name: "${{ needs.rng.outputs.value }}-slsa-attestations"
           path: "${{ needs.rng.outputs.value }}-slsa-attestations"
+
+  # cleanup deletes internal artifacts used by the delegator workflow
+  cleanup:
```

Comment:
no permissions for this? nice

Reply:
seems like no! i think it may rely on an ambient token.